cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt

```diff
@@ -2,4 +2,5 @@ psutil==5.9.4
 rfc3339>=6.2
 grpcio>=1.41.1
 googleapis-common-protos==1.6.0
-tensorflow==2.11.0
+tensorflow==2.13.0
+protobuf<=3.20.3
```

Comment:
This requirement is for the reason the same as https://github.com/kubeflow/katib/pull/2140#issuecomment-1492417056.